packages/react-core/src/components/Badge/Badge.d.ts

```diff
@@ -0,0 +1,16 @@
+import { SFC, HTMLProps, ReactNode } from 'react';
+import { Omit, OneOf } from '../../typeUtils';
+
+export const BadgeType: {
+  read: 'read';
+  unread: 'unread';
+};
+
+export interface BadgeProps extends Omit<HTMLProps<HTMLSpanElement>, 'type'> {
+  type: OneOf<typeof BadgeType, keyof typeof BadgeType>;
```

Comment:
I think type makes sense here, but I wonder if we should use `variant` since that is what is used for button and alerts.  

It also might make sense for this to just use a boolean. Unless we think there will be a tertiary state `isRead` might make more sense here.

Reply:
Thinking more I say we do a boolean. This will allow us to handle something like this: `<Badge variant="warning" isRead />`  in the future if needed.  Also read/unread seems more of a toggle to me.

Reply:
Makes sense, I went with first think that came up to my mind but boolean is much better!